docker/docker-entrypoint.sh

```diff
@@ -8,8 +8,13 @@ set -e
 DB_WAIT_TIMEOUT=${DB_WAIT_TIMEOUT-3}
 MAX_DB_WAIT_TIME=${MAX_DB_WAIT_TIME-30}
 CUR_DB_WAIT_TIME=0
-NAUTOBOT_DOCKER_SKIP_INIT=${NAUTOBOT_DOCKER_SKIP_INIT-false}
-if [ "$NAUTOBOT_DOCKER_SKIP_INIT" == "false" ]; then
+
+# lowercase NAUTOBOT_SKIP_INIT and assign to SKIP_INIT:
+SKIP_INIT=$(echo $NAUTOBOT_DOCKER_SKIP_INIT | tr '[:upper:]' '[:lower:]')
+# reassign SKIP_INIT to first character of SKIP_INIT:
+SKIP_INIT=${SKIP_INIT:0:1}
```

Comment:
This could lead to unintended consequences. Let's either expand the Boolean check below to check for `"false"` or `"f"` or ditch support for `"f"` altogether.